### ISSUE TITLE
Asset missing for Bootstrap since 4.2.1 was released which has also affected older releases on CDNJS

### ISSUE THREAD

**Issue by robinwilson16** [cdnjs/cdnjs#13174]
According to the Bootstrap repository details on CDNJS the listed resources should be available:
https://cdnjs.com/libraries/twitter-bootstrap

However the grid map file seemed to go missing when 4.2.1 was published (including for past releases) and these pages generate a 404 error:
https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap-grid.css.map
https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.2.1/css/bootstrap-grid.css.map

This leads to a compile error for projects set to update client libraries on build.
Please can the missing file be re-uploaded/or the map file excluded to avoid the error?

**Comment by PeterDaveHello** [comment]
@robinwilson16 should be the #13166 problem

**Comment by MattIPv4** [comment]
@PeterDaveHello Yeah I can access both of them linked so definitely is #13166 
Will close this to keep it all centralised.

@robinwilson16 If you get the 404 issues again on these files, please post the response headers and the URL requested to #13166 to help with us fixing this issue.

**Comment by robinwilson16** [comment]
Thanks both, it would appear it is that. On Christmas day I was getting lots of CORS issues and whilst I thought it was an issue at the remote end, it seems to fix that bit when I added CORS definitions so I proceeded to update all my code but now seems it was just a coincidence.
I can get an error on one file when publishing the project and then on another file the next time so it seems totally random. I ended up having to comment out all libman references to CDNJS and rely on existing cached copies for now until they can fix CDNJS.
